src/services/urlAssembly.ts

```diff
@@ -23,16 +21,16 @@ export function assembleUrl(route: Route, options: AssembleUrlOptions = {}): Url
   const routeQuery = assembleQueryParamValues(route.query, paramValues)
   const searchParams = combineUrlSearchParams(routeQuery, queryValues)
   const pathname = assemblePathParamValues(route.path, paramValues)
-  const hash = createHash(route.hash.value ?? options.hash).value
+  const hash = route.hash.value ? assemblePathParamValues(route.hash, paramValues) : options.hash
```

Comment:
seeing utilities named `Path` being used for hash and host weirds me out. I'm guessing they're just functionally the same?

Reply:
part of larger discussion, will remove